packages/sui-mock/README.md

```diff
@@ -0,0 +1,135 @@
+# sui-mock
+
+### Installation
+
+```sh
+npm install @s-ui/mock --save-dev
+```
+
+### Mockers
+
+#### **Request mocking with `setupMocker`**
+
+It returns all methods included in `setupWorker` and `setupServer`. It will work in browser and server sides.
+
+👉 Check `setupWorker` and `setupServer` in [MSW docs](https://mswjs.io/docs/api/).
+
+
+### Usage
+
+#### 1. Create Request handlers
+Create a `./mocks` folder in your project root and create a `index.js` file inside it.
+
+This index.js should export an array of [Request handlers](https://mswjs.io/docs/basics/request-handler).
+
+**Example:**
+
+Given, a provider endpoint handler
```

Comment:
In this case, given the request handler (GET /user) it specifies a response resolver mocked response (status 200 with body {name: 'John Doe'}).